src/Components/Notifications/ArticleFeaturedArtistNotification.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  Flex,
+  Spacer,
+  Text,
+  Box,
+  ResponsiveBox,
+  Image,
+  Button,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { FC } from "react"
+import { useFragment, graphql } from "react-relay"
+import { NotificationTypeLabel } from "Components/Notifications/NotificationTypeLabel"
+import { ArticleFeaturedArtistNotification_notification$key } from "__generated__/ArticleFeaturedArtistNotification_notification.graphql"
+import { extractNodes } from "Utils/extractNodes"
+import { FollowArtistButtonQueryRenderer } from "Components/FollowButton/FollowArtistButton"
+
+const MAX_WIDTH = 600
+
+interface ArticleFeaturedArtistNotificationProps {
+  notification: ArticleFeaturedArtistNotification_notification$key
+}
+
+export const ArticleFeaturedArtistNotification: FC<ArticleFeaturedArtistNotificationProps> = ({
+  notification,
+}) => {
+  const notificationData = useFragment(
+    ArticleFeaturedArtistNotificationFragment,
+    notification
+  )
+
+  const { headline, item } = notificationData
+
+  const article = item?.article
+  const artists = extractNodes(item?.artistsConnection)
+  const image = article?.thumbnailImage?.cropped
+
+  if (!article || !artists) {
+    return (
+      <Text variant="lg" m={4}>
+        Sorry, something went wrong.
+      </Text>
+    )
+  }
+
+  return (
+    <Box mx={4} my={4}>
+      <Flex width="100%" justifyContent="space-between">
+        <Flex flex={1}>
+          <Text fontWeight="bold" variant="xl">
```

Comment:
The in [figma](https://www.figma.com/file/PO3rjpDV29YOFR8kLI4InJ/Activity-Panel-%26-Alerts-Management?type=design&node-id=772-7261&mode=design&t=tc4NGSMnkuTRYYc3-0) headline should be just `variant="lg-display"` without `bold`

Reply:
Yeah, I've also spotted this different in other places ([here](https://github.com/artsy/force/blob/main/src/Components/Notifications/ArtworkPublishedNotification.tsx#L36) for example), but decided to follow what is already in the codebase, otherwise UI will look whimsical.

By the way the same goes for another comment from you above, I simply took [this code](https://github.com/artsy/force/blob/main/src/Components/Notifications/ArtworkPublishedNotification.tsx#L35) without looking too much into it. Is it ok if I do all these changes all-together in a separate PR?

Reply:
never mind, fixed both